assets/styles/themes/light.css

```diff
@@ -44,7 +44,10 @@ a, .breadcrumb-item.active a, .dropdown-menu i, .navigation-block i, .autocomple
     border-color: var(--primary-color);
 }
 
-.header-logo:hover, .sidebar-wrapper, .sidebar-element.separate-item div,
+.header-logo:hover {
+    background-color: var(--primary-color-dark);
+}
```

Comment:
IMO the previous version looks better, i.e. header-logo:hover background color set to --primary-color. Please consider

Reply:
Done